lib/test-setup.spec.js

```diff
@@ -10,7 +10,7 @@ before(function() {
 });
 
 beforeEach(function() {
-  this.sandbox = sinon.sandbox.create();
+  this.sandbox = sinon.createSandbox();
```

Comment:
fyi https://sinonjs.org/guides/migrating-to-5.0#sinon-is-now-a-default-sandbox

Reply:
Changes landed in `master`: https://github.com/emartech/emarsys-integration-client-js/commit/37690ff51decdc244cede05866e17c6ce5c4e900